ajax/libs/gator/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "gator",
+  "version": "1.2.4",
+  "filename": "gator.min.js",
+  "description": "Event delegation in javascript",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ccampbell/gator.git"
+  },
+  "keywords": [
+    "event",
+    "delegation"
+  ],
+  "author": "Craig Campbell",
+  "license": "Apache 2",
```

Comment:
to be recognized by our [license identifier](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json), here should use "Apache-2.0".